packages/experiments/src/components/Chiclet/Chiclet.base.tsx

```diff
@@ -14,13 +13,13 @@ export class ChicletBase extends BaseComponent<IChicletProps, IChicletState> {
   constructor(props: IChicletProps) {
     super(props);
 
-    const chicletCardProps = getOpenGraphProperties(this.props.url);
+    const chicletCardProps = this.props;
     this.state = { chicletCardProps: chicletCardProps };
```

Comment:
Since we aren't relying on `getOpenGraphProperties` anymore, I believe we actually don't need to maintain `IChicletState`. Let's delete `IChicletState` and remove these lines (16 - 17) in the constructor.

Reply:
So in the class declaration, let's do 

```tsx
export class ChicletBase extends BaseComponent<IChicletProps, {}>
```

Reply:
Actually, we won't need the constructor at all!